JustSaying.Sample.Restaurant.KitchenConsole/OrderPlacedEventHandler.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Threading.Tasks;
+using JustSaying.Messaging;
+using JustSaying.Messaging.MessageHandling;
+using JustSaying.Sample.Restaurant.Models;
+using Microsoft.Extensions.Logging;
+
+namespace JustSaying.Sample.Restaurant.KitchenConsole
+{
+    public class OrderPlacedEventHandler : IHandlerAsync<OrderPlacedEvent>
+    {
+        private readonly IMessagePublisher _publisher;
+        private readonly ILogger<OrderPlacedEventHandler> _logger;
+
+        /// <summary>
+        /// Handles messages of type OrderPlacedEvent
+        /// Takes a dependency on IMessagePublisher so that further messages can be published 
+        /// </summary>
+        public OrderPlacedEventHandler(IMessagePublisher publisher, ILogger<OrderPlacedEventHandler> log)
+        {
+            _publisher = publisher;
+            _logger = log;
+        }
+
+        public async Task<bool> Handle(OrderPlacedEvent message)
+        {
+            _logger.LogInformation("Order {orderId} for {description} received", message.OrderId, message.Description);
+
+            // TODO: handle the actual order
```

Comment:
Is this a `TODO` for you, or a note to the reader? If the latter, I'd rephrase into commentary, as otherwise it potentially makes the sample look unfinished.

Reply:
good point - done